src/redux/actions/actions.js

```diff
@@ -0,0 +1,12 @@
+import { GREETING,GOOD_BYE} from '../actionTypes/actionTypes';
```

Comment:
I'm not expecting us to have all action in one file. So you could change the name of this file to `/src/actions/homeAction.js` so everyone knows they need to follow the naming convention.

Reply:
Done